x11rb-async/examples/xclock_utc.rs

```diff
@@ -316,14 +316,14 @@ async fn main2() -> Result<(), Box<dyn std::error::Error>> {
                             && cme.window == window
                             && data[0] == atoms.wm_delete_window
                         {
-                            println!("Window was asked to close");
+                            tracing::warn!("Window was asked to close");
                             return Ok::<_, Box<dyn std::error::Error>>(());
                         }
                     }
 
-                    Event::Error(_) => println!("Unexpected error"),
+                    Event::Error(e) => tracing::warn!("Unexpected error: {e:?}"),
```

Comment:
Is this stable yet for our MSRV?

Reply:
Nope. And "that other tracing PR" had a lot of problems with that, which is why I was careful with this one. Anyway, this is just an `example/` and we do not test those on our MSRV.

Actually... we do not even test x11rb-async against the MSRV. :see_no_evil: